src/server/pkg/storage/chunk/reader.go

```diff
@@ -4,31 +4,42 @@ import (
 	"bytes"
 	"compress/gzip"
 	"context"
+	fmt "fmt"
```

Comment:
This is probably a Go Imports quirk or something? `fmt` is already named `fmt`.

Reply:
I guess? Don't know how that happened.